work/test_tts_billing.ps1

```
@@ -0,0 +1,151 @@
+}
+
+$session = New-Object Microsoft.PowerShell.Commands.WebRequestSession
+$loginResp = Invoke-Json -Method "POST" -Url "$base/api/user/login" -WebSession $session -Body @{
+  username = $adminUser
+  password = $adminPass
+}
+
+$userId = [int]$loginResp.data.id
+$authHeaders = @{ "New-Api-User" = "$userId" }
+
+$selfBefore = Invoke-Json -Method "GET" -Url "$base/api/user/self" -WebSession $session -Headers $authHeaders
```

PSScriptAnalyzer (1.25.0)

[warning] The variable 'selfBefore' is assigned but never used.

(PSUseDeclaredVarsMoreThanAssignments)